src/controllers/delivery.ts

```diff
@@ -1,28 +1,30 @@
 /* eslint-disable prefer-destructuring */
 /* eslint-disable sonarjs/no-duplicate-string */
+import {
+  isDefinedAndNotNullAndNotEmpty,
+  structuredLogger as logger,
+} from '@rudderstack/integrations-lib';
 import { Context } from 'koa';
-import { isDefinedAndNotNullAndNotEmpty } from '@rudderstack/integrations-lib';
+import { ServiceSelector } from '../helpers/serviceSelector';
+import { DeliveryTestService } from '../services/delivertTest/deliveryTest';
+import { DestinationPostTransformationService } from '../services/destination/postTransformation';
 import { MiscService } from '../services/misc';
 import {
-  DeliveryV1Response,
   DeliveryV0Response,
+  DeliveryV1Response,
   ProcessorTransformationOutput,
   ProxyV0Request,
   ProxyV1Request,
 } from '../types/index';
-import { ServiceSelector } from '../helpers/serviceSelector';
-import { DeliveryTestService } from '../services/delivertTest/deliveryTest';
-import { ControllerUtility } from './util';
-import logger from '../logger';
-import { DestinationPostTransformationService } from '../services/destination/postTransformation';
-import tags from '../v0/util/tags';
 import { FixMe } from '../util/types';
+import tags from '../v0/util/tags';
+import { ControllerUtility } from './util';
 
 const NON_DETERMINABLE = 'Non-determinable';
 
 export class DeliveryController {
   public static async deliverToDestination(ctx: Context) {
-    logger.debug('Native(Delivery):: Request to transformer::', JSON.stringify(ctx.request.body));
+    logger.info('Native(Delivery):: Request to transformer::', ctx.request.body);
```

Comment:
Won't the log file increase by huge margin with every event getting logged ?

Reply:
need to be logger.debug itself

Reply:
yes, will revert it